### PR TITLE
[DM-35243] Restore EFD backup to Sasquatch on Yagan

### PR DESCRIPTION
**Pull request by afausti**
- Update backup scripts
- Add a notebook with a procedure for incremental restores

### CODE REVIEW

File: backups/backup-chronograf.sh [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+instance=$1
+
+[ -z "$instance" ] && echo "Usage: $0 <instance name>" && exit 1
+
+export KUBECONFIG=$HOME/.kube/config-${instance}.yaml
+
+kubectl config get-contexts
+
+pod=$(kubectl get pods -n chronograf | grep chronograf | cut -d " " -f1)
```
Comment (athornton) [inline backups/backup-chronograf.sh]: The usual idiom for the last bit is `awk '{print $1}'` but cut works fine too.

File: backups/backup-influxdb.sh [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+instance=$1
+shard=$2
+
+[ -z "$instance" ] && echo "Usage: $0 <instance name>" && exit 1
+
+
+export KUBECONFIG=$HOME/.kube/config-${instance}.yaml
+
+kubectl config get-contexts
```
Comment (athornton) [inline backups/backup-influxdb.sh]: Does this line do anything?  Is it just to see if you're in the right context?  If it is then you might want to do a read from stdin before you continue, or a sleep 10 or something to let you break it before moving along.

**Review by athornton — APPROVED** [review]
It all seems reasonable.  We might at some point want to generalize it so it doesn't have to run from `${HOME}` but this should work just fine.  A couple of minor comments but nothing that should slow down moving ahead with it.

File: backups/backup-influxdb.sh (hunk 2 above)
Comment (afausti) [reply backups/backup-influxdb.sh]: Cool thanks, yes I wanted to make sure we are in the right kubectl context. I'll add a `read` to ask for confirmation there.

**Pull request merged**